libs/server/Resp/AdminCommands.cs

```diff
@@ -270,6 +270,75 @@ private bool ProcessAdminCommands<TGarnetApi>(RespCommand command, ReadOnlySpan<
                     errorCmd = "ping";
                 }
             }
+            else if (command == RespCommand.HELLO)
```

Comment:
Other commands like PING reside here. I suggest we tackle the correct command splitting in a separate PR.

Reply:
PING is implemented both here and in BasicCommands for some reason... Probably better to put this in BasicCommands in the first place, just so that people aren't confused, but up to you.

Reply:
Many non-admin commands are in this file anyway, worth a separate dev refactor.